### PR TITLE
chore: update contributing guide and the PR template

### PR DESCRIPTION
**Pull request by tatomyr**
## What/Why/How?

Streamlined the scripts usage section and updated the MR template.

## Code has been changed?

- [ ] Tested with redoc/reference-docs/workflows (internal)
- [ ] All new/updated code is covered with tests

## Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -28,8 +28,8 @@ Before submitting a pull request, please make sure the following is done:
 
 1. Fork the repository and create your branch from `main`.
 1. Run `npm install` in the repository root.
-1. If you’ve fixed a bug or added code that should be tested, don't forget to add tests!
-1. Ensure the test suite passes (`npm run test` and `npm run e2e`). Tip: to run a specific test, use this command: `npm run unit -- -t 'Test name'`. To update snapshots, run `npm run unit -- -u` for unit tests or `npm run e2e -- -u` for e2e tests.
+1. If you’ve fixed a bug or added code that should be tested, don't forget to add [tests](#tests)!
```
[the remark this answers is not in the document]
Comment (tatomyr) [reply CONTRIBUTING.md:31]: Don't they? I believe this one should lead here: https://github.com/Redocly/redocly-cli/pull/1402/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R149
At least my editor tells me it does.

[the remark this answers is not in the document]
Comment (lornajane) [reply CONTRIBUTING.md:31]: Ah, got it

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -6,9 +6,8 @@
 
 ## Screenshots (optional)
 
-## Check yourself
+## Code has been changed?
```
Comment (lornajane) [inline .github/pull_request_template.md:9]:
```suggestion
## Has code been changed?
```

**Review by lornajane — APPROVED** [review]
One tiny change for the pull request template. Thanks so much for adding this information to our guides!

**Pull request merged**